### PR TITLE
fixes #14615 - update api calls for hosts

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: lib/hammer_cli_import/base.rb [hunk 1]
```diff
@@ -427,13 +443,27 @@ def _create_entity(entity_type, entity_hash, original_id)
...
+          entity['id'] = entity['id'].to_s
+        else
+          entity_hash = {@wrap_out[entity_type] => entity_hash} if @wrap_out[entity_type]
+          debug "entity_hash: #{entity_hash.inspect}"
+          entity = mapped_api_call(entity_type, :create, entity_hash)
+        end
         debug "created entity: #{entity.inspect}"
         entity = entity[@wrap_in[entity_type]] if @wrap_in[entity_type]
-        # workaround for Bug
-        entity['id'] = entity['uuid'] if entity_type == :systems
```
Comment (beav) [inline lib/hammer_cli_import/base.rb]: I'm not all that familiar with this code. Why is it ok to remove this?

**Comment by beav** [comment]
I get the following output:

```
I, [2016-07-07T21:58:45.021201 #17150]  INFO -- : Importing from users.csv
I, [2016-07-07T21:58:45.021533 #17150]  INFO -- : Creating new organization: Satellite QE
I, [2016-07-07T21:58:46.969411 #17150]  INFO -- : Creating new organization: Organization #2
I, [2016-07-07T21:58:48.426187 #17150]  INFO -- : Creating new organization: Organization #3
I, [2016-07-07T21:58:49.809465 #17150]  INFO -- : Summary
I, [2016-07-07T21:58:49.809518 #17150]  INFO -- :   Created 3 organizations.
I, [2016-07-07T22:06:11.640302 #17607]  INFO -- : Importing from system-profiles.csv
E, [2016-07-07T22:06:11.641101 #17607] ERROR -- : Unable to import, first import content view with id 103
E, [2016-07-07T22:06:11.641310 #17607] ERROR -- : Unable to import, first import content view with id 101
E, [2016-07-07T22:06:11.641573 #17607] ERROR -- : Unable to import, first import content view with id 102
I, [2016-07-07T22:06:11.641675 #17607]  INFO -- : Summary
I, [2016-07-07T22:06:11.641714 #17607]  INFO -- :   No action taken.

```

it doesn't 500, but after running the import I don't see any hosts in my host list

File: lib/hammer_cli_import/base.rb (hunk 1 above)
Comment (thomasmckay) [inline lib/hammer_cli_import/base.rb]: yes, systems used to use the candlepin uuid as the id

**Comment by thomasmckay** [comment]
yeah, you'll need to run the 'hammer import XYZ' steps individually i think

**Comment by beav** [comment]
I ran the steps using the example export you gave me and got past the 500 error. I was not able to get all the way through since the URLs in the `repositories.csv` file didn't point to repos I had access to. I do not have commit privs on this repo but the change looks good to me.

**Pull request merged**